WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/usecases/PostBackupDownloadUseCase.kt

```diff
@@ -0,0 +1,54 @@
+package org.wordpress.android.ui.jetpack.backup.download.usecases
+
+import kotlinx.coroutines.flow.flow
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.store.ActivityLogStore
+import org.wordpress.android.fluxc.store.ActivityLogStore.BackupDownloadPayload
+import org.wordpress.android.fluxc.store.ActivityLogStore.BackupDownloadRequestTypes
+import org.wordpress.android.ui.jetpack.backup.download.handlers.BackupDownloadHandler.BackupDownloadHandlerStatus
+import org.wordpress.android.ui.utils.UiString
+import org.wordpress.android.ui.utils.UiString.UiStringRes
+import org.wordpress.android.ui.utils.UiString.UiStringText
+import org.wordpress.android.util.NetworkUtilsWrapper
+import javax.inject.Inject
+
+class PostBackupDownloadUseCase @Inject constructor(
+    private val networkUtilsWrapper: NetworkUtilsWrapper,
+    private val activityLogStore: ActivityLogStore
+) {
+    suspend fun postBackupDownloadRequest(
```

Comment:
❓ It seems this method always returns a single result and finishes. I'm wondering if using flow brings us any benefits. Wdyt?

Reply:
I was considering passing a "Starting" value, added it, then took it out at the last moment. I wanted to keep any "request errors" in the details fragment rather than the pass them along to the ProgressFragment. Honestly, I'm not sure the wizard was the way to go, but it's the course I am on so it stays. A case can be made that these are just different views in the same fragment, but I digress. Yes, I could eliminate flow from this use case